@AH-64D Apache Longbow/Addons/fza_ah64_controls/config/cfgVehicles.hpp

```diff
@@ -470,235 +471,372 @@ class CfgVehicles
-				convexComponent = "trans";
-				minimalHit = 0.5;
-				radius = 0.4;
-			};*/
-			class HitFuel
-			{
-				armor = 0.8;
+				passThrough = 0;
+			};	
+			class hit_drives_noseGearbox2	
+			{	
+				armor = 1.44 * ARMOR_SCALAR;
+				radius = HIT_RADIUS_01;
+				minimalHit = 0.1;
+				explosionShielding = 0.24;
+				name = "hit_drives_noseGearbox2";
 				material = 51;
-				name = "lfuel";
-				passthrough = 0;
-				explosionShielding=2;
-			};
-			class IrJammer
-			{
-				armor = 0.9;
+				passThrough = 0;
+			};	
+			class hit_drives_tailrotorgearbox	
+			{	
+				armor = 1.62 * ARMOR_SCALAR;
+				radius = HIT_RADIUS_01;
+				minimalHit = 0.1;
+				explosionShielding = 0.34;
+				name = "hit_drives_tailrotorgearbox";
 				material = 51;
-				name = "IR Jammer";
-				passthrough = 0.1;
-				visual = "skin_lefab";
-				minimalHit = 0.3;
-			};
-			class RfJammer: IrJammer
-			{
-				name = "RF Jammer";
-				visual = "skin_refab";
+				passThrough = 0;
+			};	
+			class hit_drives_transmission	
+			{	
+				armor = 2.88 * ARMOR_SCALAR;
+				radius = HIT_RADIUS_01;
+				minimalHit = 0.1;
+				explosionShielding = 0.30;
+				name = "hit_drives_transmission";
+				material = 51;
+				passThrough = 0;
+			};	
+			class hit_elec_battery	
+			{	
+				armor = 1.44 * ARMOR_SCALAR;
+				radius = HIT_RADIUS_02;
+				minimalHit = 0.1;
+				explosionShielding = 0.24;
+				name = "hit_elec_battery";
+				material = 51;
+				passThrough = 0;
+			};	
+			class hit_elec_generator1	
+			{	
+				armor = 0.72 * ARMOR_SCALAR;
+				radius = HIT_RADIUS_02;
+				minimalHit = 0.1;
+				explosionShielding = 0.09;
+				name = "hit_elec_generator1";
+				material = 51;
+				passThrough = 0;
+			};	
+			class hit_elec_generator2	
+			{	
+				armor = 0.72 * ARMOR_SCALAR;
+				radius = HIT_RADIUS_02;
+				minimalHit = 0.1;
+				explosionShielding = 0.09;
+				name = "hit_elec_generator2";
+				material = 51;
+				passThrough = 0;
 			};
-			class HitVRotor
+			class hit_elec_rectifier1
 			{
-				armor = 3;
+				armor = 0.72 * ARMOR_SCALAR;
+				radius = HIT_RADIUS_02;
+				minimalHit = 0.1;
+				explosionShielding = 0.10;
+				name = "hit_elec_rectifier1";
 				material = 51;
-				name = "mala vrtule";
-				passthrough = 0.1;
-				visual = "tr_blades";
-				explosionShielding = 6;
-				convexComponent = "mala vrtule";
-				minimalHit = 0.05;
-				radius = 0.06;
+				passThrough = 0;
 			};
-			class HitHRotor
+			class hit_elec_rectifier2
 			{
-				armor = 4;
+				armor = 0.72 * ARMOR_SCALAR;
+				radius = HIT_RADIUS_02;
+				minimalHit = 0.1;
+				explosionShielding = 0.10;
+				name = "hit_elec_rectifier2";
 				material = 51;
-				name = "velka vrtule";
-				passthrough = 0.1;
-				visual = "mr_blades";
-				explosionShielding = 2.5;
-				convexComponent = "velka vrtule";
-				minimalHit = 0.09;
-				radius = 0.4;
+				passThrough = 0;
 			};
-			class HitHStabilizerL1
-			{	//causes aircraft to shake slightly when damaged
-				armor = 1;
+			class hitengine1	
```

Comment:
hitEngine1/2 are the default arma names. For the other components, this would also require changes to the hit point names as well in the model. This would not be a quick fix.